pages/Login/index.jsx

```diff
@@ -127,7 +128,7 @@ class Login extends Component {
           </form>
           <div>
             <I18n.Text string="login.no_account" className={styles.noAccount} />
-            <Link href="/register" className={styles.signup}>
+            <Link href={appConfig.registerAccountUrl} className={styles.signup}>
```

Comment:
Why do we need it?

Reply:
Same explanation as before: ```This is necessary, because the /register route cant be changed to anything else at this point and it'll probably take quite some time until it will be changed. This is here to lift a blocker for us and can be useful to externals.```